### PR TITLE
velero 1.12.0 release update

### PR DESCRIPTION
**Pull request by sseago**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -161,4 +161,4 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
-replace github.com/vmware-tanzu/velero => github.com/openshift/velero v0.10.2-0.20230807193253-f0d1b0495a09
+replace github.com/vmware-tanzu/velero => github.com/openshift/velero v0.10.2-0.20230925145044-604a5fd242da
```
Comment (mateusoliveira43) [inline go.mod:164]: just curious, the last part is the last commit hash from our repo, but the `v0.10.2-0.20230925145044-` part, do you know how gets created?

Comment (sseago) [reply go.mod:164]: Yes. From the following (replace directive to konveyor-dev branch, tidy grabs the commit hash):
```
$ go mod edit -replace github.com/vmware-tanzu/velero=github.com/openshift/velero@konveyor-dev
$ go mod tidy
```

**Review by shubham-pampattiwar — APPROVED** [review]
/lgtm

File: go.mod (hunk 1 above)
Comment (kaovilai) [reply go.mod:164]: it also grabs "latest" tag to use in front for name purposes. If you are picky, you could rename v0.10.2-0. to v.1.2.0 or something else without changing the latter part and run `go mod tidy` and nothing would happen.

**Review by kaovilai — APPROVED** [review]

**Review by weshayutin — APPROVED** [review]

**Pull request merged**